plugin.json

```diff
@@ -55,7 +55,13 @@
             "key": "eventsToInclude",
             "name": "Events to include",
             "type": "string",
-            "hint": "Comma separated list of events to include"
+            "hint": "Comma separated list of events to include. If not set, all events will be sent."
```

Comment:
Hmm, looking at the code, this doesn't seem to be true, because `types.include()` always is checked even if `config.eventsToInclude` is empty.

Reply:
Yeah, not sure if it's ideal, but it's the existing behavior, so let's keep it here. Actually added "If not set, no events will be sent" to make this explicit